package.json

```diff
@@ -20,6 +20,7 @@
     "express": "^4.16.3",
     "express-jwt": "^5.3.1",
     "express-session": "^1.15.6",
+    "global": "^4.3.2",
```

Comment:
I think that you added this package per accident. The right way of installing a package globally is ` yarn global add package-name `. Please keep in mind that you can simply use `yarn add ...` to install non-global packages.

Reply:
alright, going to correct it

Reply:
thanks, corrected it